css-two-column-layout/styles.css

```diff
@@ -66,3 +66,13 @@ button {
   width: 100%;
   text-align: center;
 }
+
+.column-third {
+  flex-basis: 0;
```

Comment:
This `flex-basis` doesn't match the purpose that the name implies. What happens when you set `flex-basis: 33%`?